packages/components/src/VirtualizedList/NoRows/NoRows.component.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import theme from './NoRows.scss';
+
+function NoRows() {
+	return <div className={theme['no-result']}>No result found</div>;
```

Comment:
In fact I asked to change it to p or div, because it's a container that is placed, containing the message.
Was I wrong ? ping @frassinier